Dockerfile

```diff
@@ -39,21 +42,17 @@ ENV ZB_HOME=/usr/local/zeebe \
     ZEEBE_STANDALONE_GATEWAY=false
 ENV PATH "${ZB_HOME}/bin:${PATH}"
 
-RUN groupadd -g 1000 zeebe && \
-    adduser -u 1000 zeebe --system --ingroup zeebe
-
 WORKDIR ${ZB_HOME}
 EXPOSE 26500 26501 26502
 VOLUME ${ZB_HOME}/data
 
-COPY --chown=1000:0 docker/utils/startup.sh /usr/local/bin
-RUN chgrp 0 /usr/local/bin/startup.sh && chmod g=u /etc/passwd && chmod 0775 /usr/local/bin/startup.sh
-COPY --from=builder --chown=1000:0 /tmp/zeebe ${ZB_HOME}
-RUN chown 1000:0 -R ${ZB_HOME}/ && chmod 0775 ${ZB_HOME}/ ${ZB_HOME}/data
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
+RUN chmod g=u /etc/passwd
+RUN chown -R 1000:1000 ${ZB_HOME}
```

Comment:
The group has to be `0` for the non-root to work correctly, and I think we should set it after we copied the content, or maybe it's is even not needed, not sure how `copy --chown` works in docker out of my head.